README.rst

```diff
@@ -52,6 +52,9 @@ TOKEN
 +++++
 
 #. Open your browser's *Developer Console*.
```

Comment:
Hey @yuvipanda thanks for the contribution!

Would you mind adding a new line between these two items?  RST required a new line separating the enumerated lists, otherwise it reports an error and does not compile.
```suggestion
#. Open your browser's *Developer Console*.

```

Reply:
Thanks @rusq! I've accepted the change!

Thank you for this lovely tool!